### ISSUE TITLE
Snyk is reporting CVE-2023-7104

### ISSUE THREAD

**Issue by monwolf** [DataDog/dd-trace-go#2693]
Good afternoon,
We are using snyk to detect 3rd party vulns in our projects and we are getting a high vulnerability from your package.

gopkg.in/DataDog/dd-trace-go.v1@v1.63.1

![image](https://github.com/DataDog/dd-trace-go/assets/1116133/4d784385-39ae-4d46-bece-88b30098cf1a)

To fix it, you just have to update from go-sqlite3@v1.14.16 to go-sqlite3@1.14.18 1.14.18 

Best Regards,

**Comment by monwolf** [comment]
Hi @darccio, 

have you had time to review this issue? 

Regards,

**Comment by darccio** [comment]
@monwolf Yes, sorry, my reply slipped through the cracks. You can fix it in your side by importing v1.14.18 while we merge the update and release the next version.

**Comment by darccio** [comment]
Merged into `main`. During this week we'll start the next `dd-trace-go` release.

**Comment by darccio** [comment]
@monwolf Released as `v1.65.0`.
